.github/workflows/nightly-tests.yml

```diff
@@ -3,10 +3,10 @@ name: Nightly Tests
 on:
   schedule:
     - cron: '0 0 * * *'
-  workflow_dispatch:
```

Comment:
I think we want to retain the workflow dispatch, which enables other forks to run these tests on demand (assuming the conditions are set up similar to the integration tests conditions)